components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/SetupCmd.java

```diff
@@ -149,7 +152,7 @@ public void execute() {
         GatewayCmdUtils.setEtcd(etcd);
         LOGGER.debug("Etcd is enabled : " + isEtcdEnabled);
 
-        if(!openApi.isEmpty() || !apiName.isEmpty()){
+        if((openApi != null) || (apiName != null) || (label != null)){
```

Comment:
Is empty case handled properly?
Ex: Can we continue in case of `openApi = ""` ?

Reply:
Yes, we can continue because that issue is handled once we get into the "AddCmd" (after passing this given condition).